labs/RivePlayer/samples/RivePlayer.Samples/RivePlayer.md

```diff
@@ -17,6 +17,181 @@ subcategory: Input
+				 Source="ms-appx:///animated-login-screen.riv">
+		<Rive:NumberInput Target="numLook"
+				          Value="{x:Bind SliderRangeExample.Value, Mode=OneWay}" />
+</Rive:RivePlayer>
+<Slider x:Name="SliderRangeExample"
+        Maximum="100"
+        Minimum="0"
+        Value="0" />
+```
+
+In the example above, we nest the `Rive:NumberInput` strictly under the `Rive:RivePlayer` markup.
+
+To create a reference to the number input named `numLook` in the state machine of this Rive file, we set the `Target` property to that input name. Additionally, the `Value` property uses the `x:Bind` markup extension to bind the slider's `Value` property to the number value of the state machine input.
+
+When the slider range value is changed, the new number value will be set on the value of the `numLook` state machine input and the state machine will respond accordingly.
+
+### `<Rive:TriggerInput>`
+
+High-level class for a state machine instance of a trigger input type. This should be nested under the `Rive:RivePlayer` element in XAML
+
+### Props
+
+- `Target` - (string) name of the correlated input in the state machine
+- `x:Name` - (string) name for this trigger reference
+
+### Usage
+
+One example using Rive `TriggerInput` classes is binding it to click events of button UI Elements elsewhere in XAML. See below for a code snippet of this:
+
+```xml
+<!-- Example.xaml -->  
+<Rive:RivePlayer Width="600"
+				 Height="600"
+				 Source="ms-appx:///animated-login-screen.riv">
+		<Rive:TriggerInput Target="trigSuccess"
+						   x:Name="SuccessTriggerInput" />
+</Rive:RivePlayer>
+<Button Content="Success">
+	  <Interactivity:Interaction.Behaviors>
+			  <Interactions:EventTriggerBehavior EventName="Click">
+						<Interactions:CallMethodAction MethodName="Fire"
+												       TargetObject="{x:Bind SuccessTriggerInput}" />
+			  </Interactions:EventTriggerBehavior>
+      </Interactivity:Interaction.Behaviors>
+</Button>
+```
+
+In the example above, we nest the `Rive:TriggerInput` strictly under the `Rive:RivePlayer` markup.
+
+To create a reference to the trigger input named `trigSuccess` in the state machine of this Rive file, we set the `Target` property to that input name. Additionally, the `x:Name` property defines the name for the class so that the button knows what to bind the click event to to "fire" the trigger.
+
+When the button is clicked, it will call into the `SuccessTriggerInput` input class and invoke the `Fire`  method on the `trigSuccess` Rive trigger input where the state machine will respond accordingly.
+
+Additionally in this example, we use the [EventTriggerBehavior](https://github.com/Microsoft/XamlBehaviors/wiki/EventTriggerBehavior) and [CallMethodAction](https://github.com/Microsoft/XamlBehaviors/wiki/CallMethodAction) APIs from the [XAML Behaviors](https://github.com/Microsoft/XamlBehaviors/wiki) dependency. You can include the dependency in that example with the following snippet below in a `Dependencies.props` file:
```

Comment:
I would say "from the XAML Behaviors package"

Also, Dependencies.props isn't the only way to create a reference. Maybe say something like:

In this example we use the [EventTriggerBehavior](https://github.com/Microsoft/XamlBehaviors/wiki/EventTriggerBehavior) and [CallMethodAction](https://github.com/Microsoft/XamlBehaviors/wiki/CallMethodAction) APIs from the [XAML Behaviors](https://github.com/Microsoft/XamlBehaviors/wiki) package. You can reference this package as follows: